src/pools/PoolRouter.sol

```diff
@@ -219,14 +219,7 @@ contract PoolRouter is Auth, Multicall, IPoolRouter, IPoolRouterGatewayHandler {
             approvedAssetAmount
         );
 
-        uint128 valueChange = holdings.increase(unlockedPoolId, scId, paymentAssetId, valuation, approvedAssetAmount);
-
-        accounting.addCredit(
-            holdings.accountId(unlockedPoolId, scId, paymentAssetId, uint8(AccountType.Equity)), valueChange
-        );
-        accounting.addDebit(
-            holdings.accountId(unlockedPoolId, scId, paymentAssetId, uint8(AccountType.Asset)), valueChange
-        );
```

Comment:
IIUC, this will send the message to CV, and later CV to CP to store the increase holdings, right? Is not to much messaging? Why not update the holdings directly here?

Reply:
Ref: https://kflabs.slack.com/archives/C07PG2EUR9C/p1743689530352989?thread_ts=1743669923.076539&cid=C07PG2EUR9C

In short: This way, all holdins changes are invoked from CV. The round trips are definitely "an issue". I don't have a solution for that, so for the sake of limited time, better go with that than nothing. Maybe you have a proposal for a better way?

Reply:
> so for the sake of limited time, better go with that than nothing

Yeah, I do not want to add noise.

My proposal would be to just increase the holding in CP side just when `approveRedeem` is called. CV side processing will not send any message back, just process them and dispatch any required event. But not sure if I miss something